Makefile

```diff
@@ -324,16 +328,76 @@ copr: upload-srpms ## build in copr
 	./misc/copr-build.py https://$(SERVER)/$(REMOTE_PATH)/SRPMS/$(SRPM_BASE)
 
 #
-#	deb build
+#	release
 #
 
-deb: ## build debian package
-	./misc/gen-deb-changelog-from-git.sh
-	dpkg-buildpackage
-	# especially when building in Docker container, pull build artifact in project directory.
-	cp ../mgmt_*_amd64.deb ./
-	# cleanup
-	rm -rf debian/mgmt/
+release: $(DEB_TAR) $(RPM_TAR) $(PACMAN_TAR)
+	@echo "Creating github release..."
+	hub release create \
+		-F <( echo -e "$(VERSION)\n";echo "License: GPLv3" ) \
+		-a $(DEB_TAR) \
+		-a $(RPM_TAR) \
+		-a $(PACMAN_TAR) \
+		$(VERSION)
+
+release/mkdirs:
+	mkdir -p release/{deb,rpm,pacman}
```

Comment:
One nit pick (sorry!) Could you keep the target named release (awesome) but name the output folder `releases` please?